libevm/src/main/java/io/horizen/evm/Invocation.java

```diff
@@ -0,0 +1,30 @@
+package io.horizen.evm;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.math.BigInteger;
+
+public class Invocation {
+    public final Address caller;
+    public final Address callee;
+    public final BigInteger value;
+    public final byte[] input;
+    public final BigInteger gas;
+    public final boolean readOnly;
```

Comment:
if it doesn't involve too much refactoring (also involving changes on the SDK side) and given the fact that we don't have a getter method in the class can we rename the readOnly variable to isReadOnly just to increase readability?